app/forms/vehicles/details_form.rb

```diff
@@ -35,6 +35,10 @@ def payments_remain?
       payments_remain.to_s == "true"
     end
 
+    def no_payments_remain?
+      payments_remain.to_s == "false"
```

Comment:
Not a blocker as you have probably considered and have reason not to use but

```suggestion
      !payments_remain
```

Reply:
Nope just copied, and inverted, the method above! 😬